Index: api.go
===================================================================
--- api.go	(revision 4469)
+++ api.go	(revision 4470)
@@ -28,6 +28,29 @@
 	Bots *mux.Router // 'api/v4/bots'
 	Bot  *mux.Router // 'api/v4/bots/{bot_user_id:[A-Za-z0-9]+}'
 
+	/* Ryobi Systems CloudOffice Meeting Mod 20210325 START */
+	Meetings        *mux.Router // 'api/v4/meetings'
+	Meeting         *mux.Router // 'api/v4/meetings/{meeting_id:[A-Za-z0-9]+}'
+	MeetingByUserId *mux.Router // 'api/v4/meetings/userid/{user_id:[A-Za-z0-9]+}'
+	/* Ryobi Systems CloudOffice Meeting Mod 20210325 END */
+
